intro_to_ggraph.Rmd

```diff
@@ -0,0 +1,216 @@
+---
+output: html_document
+---
+# A brief introduction to ggraph
+
+Ying Bi and Gaoge Liu
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(ggraph)
+library(igraph)
+library(dplyr)
+library(tidygraph)
+library(tidyr)
+library(readr)
+nodes <- read_csv("dolphin_nodes.csv")
+edges <- read_csv("dolphin_edges.csv")
+```
+
+## 1 Overview
```

Comment:
For all of the following ##, delete the number followed by the ## (i.e. numbers 1-7). 
For example, change "## 1 Overview" to "## Overview".